### PR TITLE
Recompute cost price when cancel moves

### PR DESCRIPTION
**Pull request by raimonesteve**
#157595

### CODE REVIEW

**Review by albertca — COMMENTED** [review]
Why not using:

```
Product.__queue__.recompute_cost_price([product], start=date)
```

**Comment by raimonesteve** [comment]
> Why not using:
> 
> ```
> Product.__queue__.recompute_cost_price([product], start=date)
> ```

43a0de605c784433dfe5f6a7a000adf03aa1958c

File: reset.py [hunk 1]
```diff
@@ -157,6 +158,14 @@ def transition_reset(self):
                 values=['cancelled', '%s,%s' % (Reset.__name__, reset.id)],
                 where=sql_where))
 
+            products = set([m.product for m in self.confirm.moves])
+            date = min([m.effective_date or today for m in self.confirm.moves]
+                + [production.effective_date or today])
+
+            with Transaction().set_context(queue_name='production'):
```
Comment (albertca) [inline reset.py]: It's better not to use a queue_name by default. That's what's been done in core and allows for better customization.

Comment (raimonesteve) [reply reset.py]: See that production_quality_control module is already using a queue by "production" name

Comment (raimonesteve) [reply reset.py]: 287dcda50c5c01b8ce7cc89b833f784734469f23

**Review by albertca — APPROVED** [review]

**Pull request merged**